src/signature.rs

```diff
@@ -6,8 +6,8 @@ use ark_serialize::CanonicalSerialize;
 /// `SignatureComponents` contains the realized parts of a signature
 #[derive(Copy, Clone, Debug, CanonicalSerialize, CanonicalDeserialize)]
 pub struct Signature<TE: TECurveConfig + Clone> {
-    r: Affine<TE>,
-    s: TE::ScalarField,
+    pub r: Affine<TE>,
```

Comment:
There are getter methods but I can give pass anyway

Reply:
cool, returned to private param and using `.r()` method at https://github.com/kilic/arkeddsa/pull/1/commits/8efc724124d1dbf550a7d00d12ff3920851bedf2